src/coreclr/jit/importer.cpp

```diff
@@ -4911,6 +4911,89 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                 break;
             }
 
+            case NI_System_BitConverter_DoubleToInt64Bits:
+            {
+                GenTree* op1 = impPopStack().val;
+                assert(varTypeIsFloating(op1));
+
+                if (op1->IsCnsFltOrDbl())
+                {
+                    double f64Cns = op1->AsDblCon()->gtDconVal;
+                    retNode = gtNewLconNode(*(int64_t*)&f64Cns);
```

Comment:
Nit: here and below: C-style cast -> `static_cast`.

Reply:
Fixed.